.gitlab/functional_test/system_probe.yml

```diff
@@ -17,11 +17,27 @@
     AGENT_MAJOR_VERSION: 7
     DD_PIPELINE_ID: $CI_PIPELINE_ID-fnct
     CHEF_VERSION: 14.15.6
-  script:
+
+.kitchen_test_system_probe_linux:
+  extends:
+    - .kitchen_test_system_probe
+  before_script:
     - echo "CI_JOB_URL=${CI_JOB_URL}" >> $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/job_env.txt
     - echo "CI_JOB_ID=${CI_JOB_ID}" >> $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/job_env.txt
     - echo "CI_JOB_NAME=${CI_JOB_NAME}" >> $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/job_env.txt
     - echo "CI_JOB_STAGE=${CI_JOB_STAGE}" >> $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/job_env.txt
+    - cp $CI_PROJECT_DIR/minimized-btfs.tar.xz $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/minimized-btfs.tar.xz
+    - inv system-probe.test-docker-image-list > $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/docker-images.txt
+    - |
+      get_docker_secrets() {
```

Comment:
❓ Is there an advantage in making a function as you don't reuse it?
And should we make this snippet the default way to wrap-up the `aws ssm` command? It's not really uniform on all the gitlab-ci files (maybe something we should sort out in agent-platform though)

Reply:
It was mostly for the clean handling of the `xtrace` bash option.